py/emitnative.c

```diff
@@ -2697,7 +2697,7 @@ STATIC void emit_native_return_value(emit_t *emit) {
         if (return_vtype != VTYPE_PYOBJ) {
             emit_call_with_imm_arg(emit, MP_F_CONVERT_NATIVE_TO_OBJ, return_vtype, REG_ARG_2);
             #if REG_RET != REG_PARENT_ARG_RET
```

Comment:
It could be that this is supposed to be `#if REG_RET != REG_PARENT_RET` instead.

Reply:
`REG_PARENT_RET` doesn't exist, so unless I'm missing something that doesn't seem likely.

Reply:
Yes it does, in both asmxtensa.h and emitnative.c

Reply:
Ah sorry, got it mixed up with `REG_PARENT_ARG_RET`. :)